plugins/account_history_plugin/account_history_plugin.cpp

```diff
@@ -270,13 +270,13 @@ void account_history_plugin::set_program_options(options_description& cli, optio
          ("filter_on_accounts,f", bpo::value<vector<string>>()->composing(),
           "Track only transactions whose scopes involve the listed accounts. Default is to track all transactions.")
          ("get-transactions-time-limit", bpo::value<int>()->default_value(account_history_plugin_impl::DEFAULT_TRANSACTION_TIME_LIMIT),
-          "Limits the time to wait before returning for get_transactions")
+          "Limits the maximum time (in milliseconds) processing a single get_transactions call.")
          ;
 }
 
 void account_history_plugin::plugin_initialize(const variables_map& options)
 {
-   my->transactions_time_limit = options.at("get-transactions-time-limit").as<int>();
+   my->transactions_time_limit = options.at("get-transactions-time-limit").as<int>() * 1000;
```

Comment:
It is taking milliseconds and is converting to microseconds, for the comparison to time_point.count()